ksqldb-api-client/src/test/java/io/confluent/ksql/api/client/ClientTest.java

```diff
@@ -782,6 +782,31 @@ public void shouldHandleErrorResponseFromExecuteStatement() {
     assertThat(e.getCause().getMessage(), containsString("something bad"));
   }
 
+  @Test
+  public void shouldExecuteSingleStatementWithMultipleSemicolons() throws Exception {
+    // Given
+    final CommandStatusEntity entity = new CommandStatusEntity(
+        "CREATE STREAM FOO AS CONCAT(A, `wow;`) FROM `BAR`;  ",
```

Comment:
Let's also update this example to reflect proper usage of quoted strings, to avoid confusion (contributed to my question above).
```suggestion
        "CREATE STREAM FOO AS CONCAT(A, 'wow;') FROM `BAR`;  ",
```